tests/test_self_closing.py

```
@@ -0,0 +1,379 @@
+
+        for input_html, expected in cases:
+            with self.subTest(input=input_html):
+                result = self_close_void_elements(input_html)
+                self.assertEqual(result, expected)
+
+    def test_attributes_with_special_chars(self):
+        """测试属性值中包含特殊字符的情况"""
+        cases = [
+            # 属性值中有 >
+            ('<meta content="a>b">', '<meta content="a>b" />'),
+            # 属性值中有引号（转义）
```

Ruff (0.14.10)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)